src/lender/deployer.sol

```diff
@@ -105,20 +107,23 @@ contract LenderDeployer is FixedPoint {
         juniorSymbol =juniorSymbol_;
     }
 
-    function init(uint minSeniorRatio_, uint maxSeniorRatio_, uint maxReserve_, uint challengeTime_, uint seniorInterestRate_) public {
+    function init(uint minSeniorRatio_, uint maxSeniorRatio_, uint maxReserve_, uint challengeTime_, uint seniorInterestRate_, MemberlistFab memberlistFab_) public {
```

Comment:
Can we move the token name and symbol params to the init method?

Then we would have a pattern. fabs in the constructor and all other config params in init